.github/workflows/test.yml

```
@@ -268,21 +272,19 @@ jobs:
           python-version: "3.8"
       - name: Set up Rust
         uses: ./.github/actions/rust-toolchain-from-file
-      - name: venv作成
-        uses: ./.github/actions/create-venv
-      - run: pip install -r ./crates/voicevox_core_python_api/requirements.txt
+      - run: |
+          pip install --upgrade poetry
+          poetry install --with dev --with test
       - run: cargo build -p voicevox_core_c_api -vv
-      - run: maturin build --manifest-path ./crates/voicevox_core_python_api/Cargo.toml --locked
-      - run: maturin develop --manifest-path ./crates/voicevox_core_python_api/Cargo.toml --locked
+      - run: poetry run maturin build --manifest-path Cargo.toml --locked
+      - run: poetry run maturin develop --manifest-path Cargo.toml --locked
```

[manifest dir](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates)下であれば`--manifest-path`は要らないはず。

```suggestion
      - run: poetry run maturin build --locked
      - run: poetry run maturin develop --locked
```